default_payment_config.xml

```diff
@@ -229,6 +229,23 @@
         <wdoxidee_descriptor>1</wdoxidee_descriptor>
         <wdoxidee_additional_info>1</wdoxidee_additional_info>
     </payment>
+    <payment>
+        <oxid>wdpaymentoninvoice</oxid>
+        <oxactive>0</oxactive>
+        <oxtoamount>10000</oxtoamount>
```

Comment:
I noticed now that there are always the oxtoamounts added to de def config. Is this in the US or is it default from Oxid?

Reply:
This is default from Oxid.

Reply:
Thank you